Docs: added pagination information for state timeline (#91749)
grafana/grafana: docs/sources/panels-visualizations/visualizations/state-timeline/index.md

Controls how much space between rows there are. 1 = no space = 0.5 = 50% space.

### Page size (enable pagination)

> The **Page size** option lets you paginate the state timeline visualization to limit how many series are visible at once. This is useful when you have a lot of data. With paginated results, the x-axis remains static, but the y-axis dynamically changes to align with the visible data:

Reviewer: i would say this is useful when you have "a lot of series", not generally "a lot of data", which is often refers to many values.

i would also leave out mention of x axis and y axis. with pagination enabled we show a subset of all _series_ on each page.

btw, this is in contrast to Table, where we paginate rows/values. here we paginate columns/fields/series.

> The **Page size** option lets you paginate the state timeline visualization to limit how many series are visible at once. This is useful when you have a lot of data. With paginated results, the x-axis remains static, but the y-axis dynamically changes to align with the visible data:

Author: ```suggestion
The **Page size** option lets you paginate the state timeline visualization to limit how many series are visible at once. This is useful when you have many series. With paginated results, the visualization displays a subset of all series on each page:
```
